app/controllers/admin/admin_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Admin::AdminController < ApplicationController
```

Comment:
http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/ClassAndModuleChildren
Use nested module/class definitions instead of compact style
```ruby
module Admin
  class ...
  end
end
```

Reply:
```ruby
class AdminController < ApplicationController
end
```

```ruby
module Admin
  class PostsController < AdminController
  end
end